chia/server/address_manager_store.py

```diff
@@ -65,11 +66,8 @@ async def create_address_manager(cls, peers_file_path: Path) -> AddressManager:
             except Exception:
                 try:
                     # try using the new method
-                    data: Optional[bytes] = None
                     async with aiofiles.open(peers_file_path, "rb") as f:
-                        data = await f.read()
-                    assert data is not None
-                    address_manager = await cls.deserialize_bytes(data)
+                        address_manager = await cls.deserialize_bytes(io.BytesIO(await f.read()))
```

Comment:
i _think_ we could use https://docs.python.org/3/library/typing.html#typing.BinaryIO for hinting and then pass the open file directly without needing to read it into a bytesio directly.  and where we don't have a file, we could still use bytesio.  i think...